.github/workflows/CreatePWTRelease.yml

```
@@ -0,0 +1,33 @@
+name: Create PWT release
```

```suggestion
name: Create GitHub release of PWT
```

@r0b1n, can you also please add short description of this workflow?